torchci/pages/flakytest.tsx

```diff
@@ -20,71 +21,42 @@ export default function Page() {
         Suite Filter: <code>{suite === "%" ? "<any>" : suite}</code> | Test File
         Filter: <code>{file === "%" ? "<any>" : file}</code>
       </h3>
-      <em>Showing last 30 days of data.</em>
       {data === undefined ? (
         <div>Loading...</div>
       ) : (
-        (data.flakyTests as FlakyTestData[]).map((test) => {
-          const samples = data.flakySamples[getFlakyTestCapture(test)];
-          console.log(samples);
+        (data as FlakyTestInfoHUD[]).map((test) => {
           return (
-            <div key={`${test.name} ${test.suite} ${test.file}`}>
+            <div key={`${test.name} ${test.classname} ${test.file}`}>
               <h1>
-                <code>{`${test.name}, ${test.suite}`}</code>
+                <code>{`${test.name}, ${test.classname}`}</code>
               </h1>
-              from file <code>{`${test.file}`}</code>
-              <h4>Debugging instructions:</h4>
-              <p>
-                DO NOT ASSUME THINGS ARE OKAY IF THE CI IS GREEN. We now shield
-                flaky tests from developers so CI will thus be green but it will
-                be harder to parse the logs.
-              </p>
-              <p>To find relevant log snippets:</p>
-              <ol>
-                <li>
-                  Click on any of the workflow job links below, for example{" "}
-                  <a
-                    href={`https://github.com/pytorch/pytorch/runs/${test.jobIds[0]}`}
-                  >{`${test.workflowNames[0]} / ${test.jobNames[0]}`}</a>
-                </li>
-                <li>
-                  Click on the Test step of the job so that it is expanded.
-                  Otherwise, the grepping will not work.
-                </li>
-                <li>
-                  Grep for <code>{test.name}</code>
-                </li>
-                <li>
-                  There should be several instances run (as flaky tests are
-                  rerun in CI) from which you can study the logs.
-                </li>
-              </ol>
-              <div>
-                <h4> Test workflow job URLs: </h4>
-                <ul>
-                  {test.workflowNames.map((value, index) => {
-                    return (
-                      <li key={index}>
-                        <a
-                          href={`https://github.com/pytorch/pytorch/runs/${test.jobIds[index]}`}
-                        >{`${value} / ${test.jobNames[index]}`}</a>{" "}
-                        on branch {test.branches[index]}
-                      </li>
-                    );
-                  })}
-                </ul>
-              </div>
-              {samples?.length > 0 && (
-                <div>
-                  <p>Example logs: </p> <LogViewer job={samples[0]} />
-                </div>
-              )}
+              from file <code>{`${test.invoking_file}`}</code>
+              <div>Jobs:</div>
+              <ul>
+                {test.jobs.map((job) => {
```

Comment:
Looking at the screenshot on https://github.com/pytorch/test-infra/issues/4874, I think the branch name could be useful and we could consider grouping examples by that axis, i.e. just need to show few examples from one branch.  On the other hand, if it's always main branch, then that becomes a bit redundant.

Reply:
Added branch info, but I think i'll leave the ordering as is since its sorted by most recent while I think about how to make it show a larger spread of branches